docs/ABOUT.md

```diff
@@ -10,6 +10,6 @@ The most common form of Python is compiled in C; this is often invisible to the
 
 Python was started by Guido van Rossum in 1989; its name is an homage to the comedy troupe Monty Python.
 
-Python 2 is used widely but support [may end by 2020](https://www.python.org/dev/peps/pep-0373/#id2); it is highly recommended that beginners use Python 3 (they are similar, but users will encounter [some differences](http://blog.teamtreehouse.com/python-2-vs-python-3)).
+This track uses Python 3. It is highly recommended that students upgrade to at least Python 3.6 as some features used by this track may not be supported in earlier version of Python.
```

Comment:
This line seems to indicate that "3.4+" mentioned elsewhere might need tweaking?

Reply:
*highly recommended*...

My thought is that we recommend 3.6, but we still *support* 3.4. It's not EOL yet, and 3.5 is nowhere near EOL either, but when 3.6 is the default installation for Python3 on many systems now (if the OS *has* a default installation), it doesn't seem correct to suggest installing anything older.

We might amend this note to mention each new release has performance improvements over the last.

On that note, with the recent release of 3.8, should we recommend 3.7 if Python3 is not already installed?

Reply:
Generally speaking we should probably be recommending 3.7+ at the moment, 3.6+ if you're a bit out of date... nothing older. The simple fact is 3.6 has been around since before Trump assumed office. Which, admittedly, isn't _that_ long ago, but _feels_ like several decades.